ChromeExtension/style.css

```diff
@@ -0,0 +1,38 @@
+h1 { 
+  color: #2F4F4F; 
+  font-family: 'Helvetica Neue', sans-serif; 
+  font-size: 30px; 
+  font-weight: bold; 
+  letter-spacing: -1px; 
+  line-height: 1;
+}
+
+p { 
+color: #2F4F4F; 
+font-family: 'Helvetica Neue', sans-serif; 
+font-size: 15px; 
+font-weight: bold;
+text-align: justify; 
```

Comment:
nit: indent these - it's better to follow css style for readability and consistency